### ISSUE TITLE
Accessibility error in the "welcome" tag

### ISSUE THREAD

**Issue by Franqsanz** [forem/forem#12749]
Hello DEV team, I found an accessibility error in one of the tags, the "welcome" tag. I leave screenshots below:

![Captura de pantalla (1)](https://user-images.githubusercontent.com/44296203/108533144-6ab8a700-72b7-11eb-9dd1-fe6f71816025.png)
![Captura de pantalla 2021-02-19 132718](https://user-images.githubusercontent.com/44296203/108533162-70ae8800-72b7-11eb-8c7e-f35cad70bb98.png)

**Comment by cmgorton** [comment]
Thanks for reporting this!

**Comment by Franqsanz** [comment]
You're welcome :)

**Comment by ari-hacks** [comment]
👋🏾 I'd like to take this one

**Comment by cmgorton** [comment]
Thanks @ari-hacks I will assign this issue to you. Let us know if you have any questions as you start to work on it.

**Comment by jh3y** [comment]
I'll take it @cmgorton 👀 🐻 

Do we have a color in mind?

**Comment by cmgorton** [comment]
Good question. I am actually not sure. Looking at it again it looks like something maybe only the tag moderator for #welcome can change? I have admin privileges on DEV and I can't change it.

**Comment by cmgorton** [comment]
@ellativity yeah, if it is done through the admin area can you go a head and change it. We will just want to check the color contrast of whatever you change it to in dark mode.

**Comment by ellativity** [comment]
Sorry @jh3y! Sure thing, @cmgorton
